mail-filter/postforward/Manifest

```diff
@@ -0,0 +1 @@
+DIST postforward-1.1.0.tar.gz 5707 SHA256 a2968dbc6374715d503c4efba67ee587d33a045a747d74253c4dd9f43df6fffa SHA512 39d7a33c454b1529a691b24fd69b311ba368caf59168d39146654cfe319ebc3cf73835e66e6c95a43747f389a6b954ac8da0d7e934e562c16cb85dc20c0f4804 WHIRLPOOL 329c42f424497f3b6dee433eab775752c3e59d37a8139f1e9f39eef9e749971a94321b98c88c6bd76144003c39116a26cf1386b7796e913494bffa689f3f6dd5
```

Comment:
Those are wrong hashes. Please remove the Manifest file and make `repoman manifest` regenerate them.